CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+# [5.0]
+
+- **BREAKING CHANGE** Removed `customPropertyValue` mixin
```

Comment:
might be good to add a note that consumers should remove usage entirely and use a build pipeline that will automatically transpile css custom properties, e.g. PostCSS - just be clear that there is no replacement/'new interface' for what it was doing